### ISSUE TITLE
v2 Binaries?

### ISSUE THREAD

**Issue by anaday** [liquidcitymotors/Via-for-VCVRack#34]
I can no longer build from the source code. Not compatible with my system OSX 10.3.6 High Sierra. I'm having the same issue with TRS modules. Are you planning on releasing these? And will they work on systems less than 10.14? Thanks.

**Comment by liquidcitymotors** [comment]
Hey @anaday, I'd like to make these compatible with yours and other old MacOS systems if possible. I have an old iMac that can't get past 10.11.6 that I can start testing on.

Would you like to try binary builds from my MacOS system to see if they work for you? 

For the record, I am planning on releasing at least the Via modules for v2 into the plugin library; TRS is so likely to change majorly that it would likely break between versions and probably needs a lot more work before its ready for a library release.

**Comment by anaday** [comment]
That would be great! I would definitely like to test them out for you. Thanks so much!

**Comment by liquidcitymotors** [comment]
Hi @anaday,

I also hit some snags when building on Mac with the one 3rd party library that is (unnecessarily) included in the starling-dsp submodule. I yeeted that for now and built this:

[Starling_Via.zip](https://github.com/starlingcode/Via-for-Rack/files/8653464/Starling_Via.zip)

Can you install with that and if so, do things seem to work?

**Comment by anaday** [comment]
@liquidcitymotors ,

The plugin loads fine however, Sync3XL crashes Rack when adding to a patch. Everything else (including Sync3L expander) works fine. I'll load some patches later to test some more and let you know what I find.

Here's the crash log:
[crashlog.txt](https://github.com/starlingcode/Via-for-Rack/files/8653973/crashlog.txt)

Thanks

**Comment by liquidcitymotors** [comment]
Thanks @anaday!

Before you spend too much time investigating the crash, it might be wise for me do some investigation because I barely tested this build myself. I'll let you know what I come up with once I have a chance to really dig in.

**Comment by liquidcitymotors** [comment]
As expected there was a silly mistake with the path for loading the sync3xl scale file, this should work much better @anaday

[Starling_Via.zip](https://github.com/starlingcode/Via-for-Rack/files/8665027/Starling_Via.zip)

**Comment by anaday** [comment]
Hey @liquidcitymotors ,

Everything looks pretty good so far. They all load, unload, reload fine. I haven't encountered any problems with Gateseq, Sync3, Meta and Osc3. The others, I don't have much experience with, so I'll take a good look at those over the weekend using the documentation.

Thanks again for work on these.

Edit: Everything works great. Checked for crashes when placing modules, opening and closing patches, etc. Parameter and port hints look good. I was able to test Sync3XL and its expander and love this. Awesome work!
